scanners/sslyze/parser/parser.test.js

```diff
@@ -408,7 +434,10 @@ test("parses an empty result file correctly", async () => {
   await expect(validateParser(findings)).resolves.toBeUndefined();
   expect(findings).toEqual([]);
 });
+*/
 
+// Commented out until I can reproduce the test files
+/*
```

Comment:
Same question here: Can this be re-added or deleted?

Reply:
I don't know how to reproduce the mixed-connectivity test file. So I can't readd it just yet. Can you help with that ?

Reply:
Not sure about this, either. Maybe @J12934 knows more once he's back.

Reply:
I think after our discussion with @J12934, you can just remove this test case and associated files for the moment.

Reply:
I don't know why I thought i already did this, but it's done now ^^ .